### DISCUSSION TITLE
Issues debt and workflow

### DISCUSSION THREAD

**Discussion by alexxa** [sap-linuxlab/community.sap_install#580] [General]
Currently, there are 79 issues open on this project. Some of them are like a question asked and answered, but that issue remains open. Others have merged PR mentioned, but that issue is not closed - does it mean an issue still need more work or smth else? Overall, that many issues may indicate some project problems:

 - **bugs and tech debt** within the codebase, which only pile up over time
 - **incomplete features**
 - **lack of maintenance** or **overwhelmed development** due to limited resources, as a small team or lack of time
  
Furthermore, when users see a big number of open issues on the repo, they may have various thoughts, e.g:
 
- **concerns about stability and reliability**
- **doubts about project health**, how actively project is maintained and supported
- **expectations for timely resolutions** if issues are reported
- **decision to contribute**

So, here is a kind suggestion to deal with a big number of currently open issues, e.g: 

 - triage existing, close resolved and inactive, label them, assign, etc
 - match the release schedule to RHEL, prioritize and label issues accordingly and commit to that development cycle
 - add [workflow to automate issues handling](https://docs.github.com/en/actions/managing-issues-and-pull-requests) (triage label on new, warning about inactive, backlog, close inactive, etc) - I will be happy to implement it.

**Comment by alexxa** [comment]
With @berndfinger we briefly touched on (enhancing) labeling schema. Please take a look at this [three level labeling schema](https://medium.com/@dave_lunny/sane-github-labels-c5d2e6004b63)  as an option.

**Reply by Wabri** [reply]
I dont know about 3 level labeling schema, there are something that I don't like about it. The "Status" label is not really useful for me on our repositories, but I'm ok with the "Priority" and "Type" labels.

I think a better method is to use the github feature like the "assign issue to", so if the issue is not assigned to none signify that is not a wip. Same for the PR, if a PR have "assignees" only that person have the permission to merge the PR. We can also add a minimun reviewers on PR in order to merge only the one that have more than one or two approves.

The automation around the labeling, at least for me, is something not required, we can be the proxy for ourself for the number of new issue we have per day.
